pkg/codegen/schema/schema.go

```diff
@@ -70,6 +71,8 @@ func (t primitiveType) String() string {
 		return "pulumi:pulumi:Asset"
 	case jsonType:
 		fallthrough
+	case anyResourceType:
+		fallthrough
```

Comment:
Curious, why not `return "pulumi:pulumi:Resource"`?

Reply:
Looks like an oversight, I think it should be a return here.
```suggestion
	case anyResourceType:
		return "pulumi:pulumi:Resource"
```